icicle-compiler/main/icicle.hs

```diff
@@ -194,6 +196,20 @@ pFlagDrop =
     long "drop-to-output" <>
     help "write partial results to dropped-X.txt or normal output"
 
+pZebraChunkSize :: Parser ZebraChunkSize
+pZebraChunkSize =
+  flip option (long "zebra-chunk-size" <> value defaultZebraChunkSize) $
+    readerAsk >>= \s -> case readMaybe s of
+      Just i  -> return (ZebraChunkSize i)
+      Nothing -> readerError "--chunk-size NUMBER_BYTES"
+
+pZebraAllocLimit :: Parser ZebraAllocLimit
+pZebraAllocLimit =
+  flip option (long "zebra-alloc-limit" <> value defaultZebraAllocLimit) $
+    readerAsk >>= \s -> case readMaybe s of
+      Just i  -> return (ZebraAllocLimit i)
+      Nothing -> readerError "--zebra-alloc-limit NUMBER_BYTES"
```

Comment:
I think bytes is too fine-grained.. in zebra I parsed a double and used that as gigabytes. I also kind of like having the measure in the argument: eg `--zebra-alloc-limit-gb 2`.

just a suggestion